lib/Metadata.php

```diff
@@ -44,6 +44,11 @@ class Metadata
     private $fieldMappings = [];
 
     /**
+     * @var bool
+     */
+    private $removeLive = true;
```

Comment:
@danrot in my opinion, the `syncRemovLive` sounds good but not perfect ... but i cannot think of any other name. what should i do? (: